include/swift/SIL/GraphFunctionDeviceInfo.h

```diff
@@ -156,6 +203,8 @@ static inline std::string getDeviceString(DeviceId deviceId) {
     return TF_DEFAULT_TPU_DEVICE;
   case DeviceType::ALL:
     return TF_ALL_DEVICES;
+  case DeviceType::RUNTIME:
+    return "";
```

Comment:
Should we return "RUNTIME" or something along those lines?

Reply:
The returned string is expected to form a valid TF device string (see API comment). 

We would return "RUNTIME" and let caller interpret it appropriately. But I feel returning "" would be simpler. Let me know if you have a strong preference.

Reply:
Discussed in the other comment. So closing this one.